cmd/ci-operator-prowgen/main.go

```diff
@@ -563,7 +566,16 @@ func generateJobs(
 		}
 
 		if element.Cron == nil {
-			presubmits[orgrepo] = append(presubmits[orgrepo], *generatePresubmitForTest(element.As, info, label, podSpec, true, configSpec.CanonicalGoRepository))
+			var alwaysRun bool
+			switch element.PullRequestPolicy {
+			case "Always", "":
+				alwaysRun = true
+			case "OnRequest":
```

Comment:
` RunIfChanged` makes this more complicated, no? `RunIfChanged` implies `alwaysRun = false`, and also nonempty `PullRequestRunIfChanged` and empty `PullRequestPolicy` together imply `alwaysRun = false` too.